packages/cli/src/lib/commands/space.ts

```diff
@@ -78,4 +78,29 @@ export const space: CommandDefinition = (program) => {
 
             await displayStream(await managerClient.getLogStream());
         });
+
+    const accessKeyCmd = spaceCmd.command("accessKey");
```

Comment:
the description is missing:
![image](https://user-images.githubusercontent.com/61054769/223117388-dbb6c290-8404-479f-88b9-dd4c5fff6968.png)

maybe smth like:
```suggestion
    const accessKeyCmd = spaceCmd
        .command("accessKey")
        .description("Manage space accessKey");
```

?

Reply:
I wonder If we'll extend access key purpose beyond Self Hosted Hubs connections 🤔